### PR TITLE
Added solution for 392_is_subsequence

### PR DESCRIPTION
**Pull request by shekhuverma**
Added solution for 392_is_subsequence problem from Leetcode

### CODE REVIEW

File: python/392_Is_Subsequence.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+	def isSubsequence(self, s: str, t: str) -> bool:
+		for a in s:
+			if a in t:
+				for b in range(0, len(t)):
+					if a==t[b]:
+						t=t[b+1:]
+						break
+			else:
+				return(False)
+		return(True)
```
Comment (qiyuangong) [inline python/392_Is_Subsequence.py]: Add an empty line at the end of source file

File: python/392_Is_Subsequence.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class Solution:
+	def isSubsequence(self, s: str, t: str) -> bool:
+		for a in s:
```
Comment (qiyuangong) [inline python/392_Is_Subsequence.py]: Pay attestation to code style. For example, using 4 space instead of tab.

**Comment by shekhuverma** [comment]
I have fixed the code stlying issues.

**Review by qiyuangong — APPROVED** [review]
LGTM

**Pull request merged**